src/util/Injected.js

```diff
@@ -556,48 +581,127 @@ exports.LoadUtils = () => {
         return msg;
     };
 
+    window.WWebJS.getChat = async (chatId, { getAsModel = true } = {}) => {
+        const isChannel = chatId.match(/@(.+)/)[1] === 'newsletter';
+        let chat;
 
-    window.WWebJS.getChatModel = async chat => {
+        if (isChannel) {
+            chat = window.Store.NewsletterCollection.get(chatId);
+            if (!chat) {
+                await window.Store.ChannelUtils.loadNewsletterPreviewChat(chatId);
+                chat = await window.Store.NewsletterCollection.find(window.Store.WidFactory.createWid(chatId));
+            }
+        } else {
+            const chatWid = window.Store.WidFactory.createWid(chatId);
+            chat = window.Store.Chat.get(chatWid) ?? await window.Store.Chat.find(chatWid);
```

Comment:
I think node 12 not support this

Reply:
@shirser121 
> I think node 12 not support this

You are right, I [fixed](https://github.com/pedroslopez/whatsapp-web.js/pull/2620/commits/f443778172df9ebc81ad06218e318b2051b48eaa) the PR to support the Node v12